extensions/oidc/runtime/src/main/java/io/quarkus/oidc/runtime/CodeAuthenticationMechanism.java

```diff
@@ -490,6 +490,20 @@ && isRedirectFromProvider(context, configContext)) {
                         codeFlowParams.append(AMP).append(OidcConstants.TOKEN_SCOPE).append(EQ)
                                 .append(OidcCommonUtils.urlEncode(String.join(" ", scopes)));
 
+                        MultiMap requestQueryParams = null;
+                        if (!configContext.oidcConfig.getAuthentication().forwardParams.isEmpty()) {
+                            requestQueryParams = context.queryParams();
+                            for (String forwardedParam : configContext.oidcConfig.getAuthentication().forwardParams.get()) {
+                                if (requestQueryParams.contains(forwardedParam)) {
+                                    for (String requestQueryParamValue : requestQueryParams.getAll(forwardedParam))
+                                        codeFlowParams.append(AMP).append(forwardedParam).append(EQ)
+                                                .append(OidcCommonUtils
+                                                        .urlEncode(String.join(" ", requestQueryParamValue)));
```

Comment:
Why are you calling `String.join` here?

Reply:
@gastaldi Silly copy and paste :-), let me fix it

Reply:
@gastaldi Fixed, sorry about that